rbac/ledger_sync/inbound/rbac_transactions.py

```diff
@@ -274,7 +275,7 @@ def fetch_role_relationships(role_id, conn):
     return result
 
 
-def delete_user_transaction(inbound_entry, user_in_db, key_pair):
+async def delete_user_transaction(inbound_entry, user_in_db, key_pair):
```

Comment:
Because these functions are now async, this is causing LDAP inbound delta sync tests to fail. Look at Vamshi's code and see where he added these async create transaction functions.